Makefile

```diff
@@ -0,0 +1,4 @@
+CC=gcc
+
+accuraterip-checksum: accuraterip-checksum.c
+	$(CC) -std=c99 -l sndfile -o $@ $<
```

Comment:
Thanks for your work! :)
Will merge as soon as the following issues are resolved:

1. Which make tool is this supposed to be written for? If I knew that, I would consult the manpage instead of asking you the following questions :)

2. I suppose this ought to be a shell commandline. If yes, then notice this pitfall:
http://mywiki.wooledge.org/BashPitfalls#for_arg_in_.24.2A
I.e. it should instead be:
```$(CC) -std=c99 -l sndfile -o "$@" ...```

3. What is the ```$<``` supposed to do? Google and DuckDuckGo don't seem to accept this as a search query, so perhaps tell me the literal description of this shell variable so I can search for documentation. Or point me to the documentation right away.